src/test/shared/sam/cli/samCliInit.test.ts

```diff
@@ -164,4 +166,36 @@ describe('runSamCliInit', async () => {
 
         await runSamCliInit(sampleSamInitArgs, context)
     })
+
+    it('Passes --app-template if version >= 0.30.0', async () => {
+        const processInvoker: SamCliProcessInvoker = new ExtendedTestSamCliProcessInvoker(
+            (spawnOptions: SpawnOptions, args: any[]) => {
+                assertArgIsPresent(args, '--app-template')
```

Comment:
should these be combined with `assertArgsContainArgument`?

Reply:
@rli is correct. Good catch.